utils/h2ph.PL

```diff
@@ -851,6 +877,37 @@ DEFINE
 		    quotemeta($define{$_}), "\" } }\n\n";
 	    }
 	}
+
+	print PREAMBLE <<SIZEOF;
+%sizeof = (
+	char	=> $Config{charsize},
+	uchar	=> $Config{charsize},
+	u_char	=> $Config{charsize},
+
+	short	=> $Config{shortsize},
+	ushort	=> $Config{shortsize},
+	u_short => $Config{shortsize},
+
+	int	=> $Config{intsize},
+	uint	=> $Config{intsize},
+	u_int	=> $Config{intsize},
+
+	long	=> $Config{longsize},
+	ulong	=> $Config{longsize},
+	u_long	=> $Config{longsize},
+
+	FILE	=> $Config{ptrsize},
+	key_t	=> $Config{ptrsize},
+	caddr_t	=> $Config{ptrsize},
+
+	float	=> $Config{intsize},
```

Comment:
This could be `length(unpack("f", 0.0))`

Reply:
Did you mean `pack`?

Reply:
Oops, yes.